lectures/time_series_with_matrices.md

```diff
@@ -0,0 +1,695 @@
+
+You can read about the details of that model in [this](https://python.quantecon.org/samuelson.html)
+QuantEcon lecture.
+
+(That lecture also describes some technicalities about second-order linear difference equations.)
+
+In this lecture, we'll also learn about an **autoregressive** representation and a **moving average** representation of a  non-stationary
+univariate time series $\{y_t\}_{t=0}^T$.
+
+We'll also study a "perfect foresight" model of stock prices that involves solving
+a "forward-looking" linear difference equation.
+
+We will use the following imports:
+
+```{code-cell} ipython
+import numpy as np
+%matplotlib inline
+import matplotlib.pyplot as plt
+from matplotlib import cm
+plt.rcParams["figure.figsize"] = (11, 5)  #set default figure size
+```
+
+## Samuelson's model
+
+Let $t = 0, \pm 1, \pm 2, \ldots$ index time.
+
+For $t = 1, 2, 3, \ldots, T$ suppose that
+
+```{math}
+:label: tswm_1
+
+y_{t} = \alpha_{0} + \alpha_{1} y_{t-1} + \alpha_{2} y_{t-2}
+```
+
+where we assume that $y_0$ and $y_{-1}$ are given numbers
+that we take as **initial conditions**.
+
+In Samuelson's model, $y_t$ stood for **national income** or perhaps a different
+measure of aggregate activity called **gross domestic product** (GDP) at time $t$.
+
+Equation {eq}`tswm_1` is called a **second-order linear difference equation**.
+
+But actually, it is a collection of $T$ simultaneous linear
+equations in the $T$ variables $y_1, y_2, \ldots, y_T$.
+
+```{note}
+To be able to solve a second-order linear difference
+equation, we require two **boundary conditions** that can take the form
+either of two **initial conditions** or two **terminal conditions** or
+possibly one of each.
+```
+
+Let’s write our equations as a stacked system
+
+$$
+\underset{\equiv A}{\underbrace{\left[\begin{array}{cccccccc}
+1 & 0 & 0 & 0 & \cdots & 0 & 0 & 0\\
+-\alpha_{1} & 1 & 0 & 0 & \cdots & 0 & 0 & 0\\
+-\alpha_{2} & -\alpha_{1} & 1 & 0 & \cdots & 0 & 0 & 0\\
+0 & -\alpha_{2} & -\alpha_{1} & 1 & \cdots & 0 & 0 & 0\\
+\vdots & \vdots & \vdots & \vdots & \cdots & \vdots & \vdots & \vdots\\
+0 & 0 & 0 & 0 & \cdots & -\alpha_{2} & -\alpha_{1} & 1
+\end{array}\right]}}\left[\begin{array}{c}
+y_{1}\\
+y_{2}\\
+y_{3}\\
+y_{4}\\
+\vdots\\
+y_{T}
+\end{array}\right]=\underset{\equiv b}{\underbrace{\left[\begin{array}{c}
+\alpha_{0}+\alpha_{1}y_{0}+\alpha_{2}y_{-1}\\
+\alpha_{0}+\alpha_{2}y_{0}\\
+\alpha_{0}\\
+\alpha_{0}\\
+\vdots\\
+\alpha_{0}
+\end{array}\right]}}
+$$
+
+or
+
+$$
+A y = b
+$$
+
+where
+
+$$
+y = \begin{bmatrix} y_1 \cr y_2 \cr \vdots \cr y_T \end{bmatrix}
+$$
+
+Evidently $y$ can be computed from
+
+$$
+y = A^{-1} b
+$$
+
+The vector $y$ is a complete time path $\{y_t\}_{t=1}^T$.
+
+Let’s put Python to work on an example that captures the flavor of
+Samuelson’s multiplier-accelerator model.
+
+We'll set parameters equal to the same values we used in [this QuantEcon lecture](https://python.quantecon.org/samuelson.html).
```

Comment:
Since the lecture ``samuelson`` is not in the intro series (but in the intermediate series), I replace this with the website link instead.